src/include/optimizer/stats/histogram.h

```diff
@@ -100,7 +103,7 @@ class Histogram {
       // -1 because we want index to be element less than b
       i = std::abs(i + 1) - 1;
     }
-    assert(i >= 0 && i < bins.size() - 1);
+//    assert(i >= 0 && i < bins.size() - 1);
```

Comment:
There is a compile error.
![image](https://cloud.githubusercontent.com/assets/5547711/24887362/1c777bfc-1e29-11e7-99c1-eaf23639c95c.png)